sema/type.go

```diff
@@ -1615,324 +1615,293 @@ func (*FixedPointNumericType) CheckInstantiated(_ ast.HasPosition, _ common.Memo
 
 // Numeric types
 
-var (
```

Comment:
gofmt is acting up here when using method call chains in a `var` block: it applies the correct indentation for the wrapped method calls, then undoes the indentation for whatever reason.

Changing the large `var` block to individual var declarations (automatically refactored using Goland) makes gofmt produce the correct indentations and makes the linter happy